about.html

```diff
@@ -0,0 +1,167 @@
+            <div class="container">
+                <p>Technology is the term used to describe the application of scientific knowledge in practical situations.
+                    With this explanation,
+                    our students are able to understand whats is about and as well as software development.
+                    It is important to know that Technology increases the utility of products and services and contributes
+                    to value creation.</p>
+            </div>
+            <div class="img">
+                <img src="images/cc-global.png" alt="Tech-Fair">
+            </div>
+        </section>
+
+        <section class="blog">
+            <h2 class="course gray">The Right Way to Learn Technology</h2>
+            <div class="container">
+                <p class="blog-ld">Technology at 21st century has been best define for humanity.</p>
+                <div class="flex">
+                    <div class="image-list">
+                        <img src="images/blog/portfolio-01.jpg" alt="course-blog">
+                        <div class="content">
+                            <h3>2021</h3>
+                            <p>2021 Technology Fair summit in New York</p>
+                        </div>
+                    </div>
+                    <div class="image-list">
+                        <img src="images/blog/portfolio-02.jpg" alt="course-blog">
+                        <div class="content">
+                            <h3>2022</h3>
+                            <p>2022 Technology training workshop for Girls in Ghana</p>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </section>
+    </course>
+    
+
+    <section class="partner about-partner">
+        <h2 class="course white">BootCamp Patner</h2>
+        <div class="container">
+            <div class="card-img">
+                <img src="images/cur-part/mozilla.svg" alt="Firefox">
+            </div>
+            <div class="card-img">
+                <img src="images/cur-part/google.svg" alt="Google">
+            </div>
+            <div class="card-img">
+                <img src="images/cur-part/naver.svg" alt="Naver">
+            </div>
+            <div class="card-img">
+                <img src="images/cur-part/daumkakao.svg" alt="Daumkakao">
+            </div>
+            <div class="card-img">
+                <img src="images/cur-part/airbnb.svg" alt="Airbnb">
+            </div>
+        </div>
+    </section> 
+    </course>
+
+    <!----- Footer section ----->
+     <footer class="footer about-footer">
+        <div class="container grid">
+            <img src="images/cck-logo1.png" alt="course-logo">
+
+            <div class="korea-text">
+                <p>2015 Creative Commons Korea. Some Rights Reserved.</p>
+                <p>달리 정하지 않은 웹사이트의 모든 내용은 크리에이티브 커먼즈 저작자표시 2.0 대한민국 라이선스에 따라 이용할 수 있습니다.</p>
+             </div>
+        </div>
+    </footer>
```

Comment:
- [ ] As I mentioned above :point_up: please stick to the Behance design your footer in the mobile version is not the same as the Behance design :+1: check below :point_down: the comparison between  your design and Behance design :slightly_smiling_face: 

**Your Design**
![Screenshot from 2023-05-27 00-05-31](https://github.com/evansnyamekye/Capstone-PR/assets/45226848/1e51556f-9860-401b-90ef-8e41f8fe5b8b)

**Behance Design**
![Screenshot from 2023-05-27 00-05-43](https://github.com/evansnyamekye/Capstone-PR/assets/45226848/b2c609e3-ea9f-4851-8675-7f9889dc70f6)